src/test/java/com/fasterxml/jackson/dataformat/xml/ser/TestCharset.java

```diff
@@ -0,0 +1,40 @@
+package com.fasterxml.jackson.dataformat.xml.ser;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlTestBase;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+public class TestCharset extends XmlTestBase
```

Comment:
Good start, probably will want to check Latin-1 (ISO-8859-1) as well.

I guess this is WIP so probably were already planning (ditto for basic UTF-8)

Reply:
Sure, I can add more tests, including for other charsets. I guess that the point at this stage is try to agree an API approach first. Keeping the number of test cases low at this point means it's easier for me to adjust the API (without the need to then rewrite all the tests).

Reply:
Good point, agreed.